backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlSessionFieldFactory.java

```diff
@@ -30,10 +31,13 @@ public GraphQLFieldDefinition signoutField(Database database) {
         .type(GraphqlApiMutationResult.typeForMutationResult)
         .dataFetcher(
             dataFetchingEnvironment -> {
-              String user = database.getActiveUser();
-              database.setActiveUser(GraphqlConstants.ANONYMOUS);
+              MolgenisSessionManager sessionManager =
```

Comment:
this is how we can pass a parameter on each request, in this case we wrapped the ctx